src/SDKs/_metadata/azsadmin_resource-manager_azurebridge.txt

```diff
@@ -0,0 +1,5 @@
+﻿Executing AutoRest command
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/azsadmin/resource-manager/azurebridge/readme.md --csharp --csharp-sdks-folder=E:\github\azure-sdk-for-net\src\AzureStack\AzureBridgeAdmin\AzureBridge.Admin --version=latest --reflect-api-versions
+
```

Comment:
Please generate the code using generate.ps1 or generate.cmd and commit those changes.
Looks like autorest failed to generate the code from config file here